src/main.ts

```diff
@@ -1,14 +1,14 @@
-import axios from 'axios';
-import OauthClient from '@girder/oauth-client';
 import * as Sentry from '@sentry/vue';
 import Vue from 'vue';
 import '@/plugins/composition';
+// import DatetimePicker from 'vuetify-datetime-picker';
 import Cesium from './plugins/cesium';
 import App from './App.vue';
 import router from './router';
 import vuetify from './plugins/vuetify';
 import { restoreLogin, oauthClient, axiosInstance } from './api/rest';
 
+// Set token to `null` to avoid warning
 Cesium.Ion.defaultAccessToken = process.env.VUE_APP_CESIUM_ACCESS_TOKEN;
```

Comment:
```suggestion
Cesium.Ion.defaultAccessToken = process.env.VUE_APP_CESIUM_ACCESS_TOKEN;
```
```suggestion
Cesium.Ion.defaultAccessToken = null;
```

Reply:
I was originally thinking of keeping this and then updating the readme but on second thought it probably is better to change it here.